specification/powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/enterprisePolicy.json

```diff
@@ -441,6 +441,10 @@
               "description": "Key vault properties."
             }
           }
+        },
```

Comment:
This should not be here as systemData is required top level property for azure resource.

Reply:
Done.